### PR TITLE
Technical Review - Borehole Design

### PR DESCRIPTION
**Pull request by KellyVance**

### CODE REVIEW

File: vocabularies/borehole-design.ttl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/borehole-design> a owl:Ontology .
+
+bhd:conceptScheme a skos:ConceptScheme ;
+    dct:created "2019-10-03T16:24:51+10:00"^^xsd:dateTime ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:modified "2019-10-03T16:32:36+10:00"^^xsd:dateTime ;
+    dct:source "Developed by the Geological Survey of Queensland." ;
+    skos:definition "A description of the intended configuration and trajectory of a borehole"@en ;
```
Comment (LukeHauck) [inline vocabularies/borehole-design.ttl]: Insert ending full stop.

File: vocabularies/borehole-design.ttl [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    skos:hasTopConcept bhd:deviated,
+        bhd:inclined,
+        bhd:lateral,
+        bhd:vertical ;
+    skos:prefLabel "Borehole Inclination Design"@en .
```
Comment (LukeHauck) [inline vocabularies/borehole-design.ttl]: Does this need to be consistent with the name of the file?
_"borehole-inclination-design.ttl"_ or _prefLabel is "Borehole Design"_

File: vocabularies/borehole-design.ttl (hunk 1 above)
Comment (KellyVance) [reply vocabularies/borehole-design.ttl]: Changed

File: vocabularies/borehole-design.ttl (hunk 2 above)
Comment (KellyVance) [reply vocabularies/borehole-design.ttl]: Changed. Forgot to change the prefLabel when i changed the file name and URIs.

**Pull request merged**